src/Main.cpp

```diff
@@ -96,6 +96,20 @@ int main(int argc, char** argv)
         }
         std::cout << wp_device.getConfig() << std::endl;
     }
+    else if (cmd == "set_id") {
+        if (argc != 7)
+        {
+            std::cerr << "wrong number of arguments" << std::endl;
+        }
+        power_whisperpower::Device device(device_group, device_id);
+        waitResult(*can_device, device, device.querySerialNumber());
+        uint32_t serial_number = device.getSerialNumber();
+        int device_new_id(std::stoi(argv[6]));
+        waitResult(*can_device, device, device.querySetId(device_new_id, serial_number));
```

Comment:
we verified in live that we should send the set id query to can id 0x600 (broadcast)